### DISCUSSION TITLE
Playmemories?

### DISCUSSION THREAD

**Discussion by daevidt** [KillerInk/FreeDcam#213] [General]
Hello,

In the Fdroid description "PlayMemories (Sony ILC WIFI Remote API)" is listed as supported. I can't really find anything about it in the app, but I would love this feature as I have such a camera. Is it really supported? How can I use it? Or maybe it is accidentally in the description?

Thanks,

David

**Comment by KillerInk** [comment]
Hi
sorry its no longer included.
it got dropped due a code rework and also my QX1 died.
thx for the note. i forgot to update the fastlane description

greets
